pkg/apis/cluster/v1alpha1/types.go

```diff
@@ -116,6 +137,100 @@ type ClusterSpec struct {
 	// any resource that does not tolerate the Taint.
 	// +optional
 	Taints []corev1.Taint `json:"taints,omitempty"`
+
+	// ResourceModels is the list of resource modeling in this cluster. Each modeling quota can be customized by the user.
+	// Modeling name must be one of the following: cpu, memory, storage, ephemeral-storage.
+	// If the user does not define the modeling name and modeling quota, it will be the default model.
+	// The default model grade from 0 to 8.
+	// When grade = 0 or grade = 1, the default model's cpu quota and memory quota is a fix value.
+	// When grade greater than or equal to 2, each default model's cpu quota is [2^(grade-1), 2^grade), 2 <= grade <= 7
+	// Each default model's memory quota is [2^(grade + 2), 2^(grade + 3)), 2 <= grade <= 7
+	// E.g. model 0 is like this:
```

Comment:
Much clear now. Should we also describe the last grade, since we are using `MAXINT`?

> E.g. model 0 is like this:

`model 0`?

Reply:
I modify to grade now. Done.